src/components/calendar/Calendar.tsx

```diff
@@ -33,6 +35,7 @@ export interface ICalendarOwnProps extends React.ClassAttributes<Calendar> {
     startingDay?: number;
     selectionRules?: ICalendarSelectionRule[];
     isLinkedToDateRange?: boolean;
+    simple?: boolean;

```

Comment:
can you have a default prop `simple` set to false ? will it not break every existing usage of the component otherwise ?

Reply:
simple is already set to undefined by default so everywhere it isn't set, it won't be evaluated to simple.

Reply:
indeed, using simple: false everywhere in https://github.com/coveo/react-vapor/pull/653/files#diff-d31af32b280c5ab5caa1d211f11b91fa fooled me, sorry for that

Reply:
That is in the date picker state where it is set to false by default.